runtime/testy/Ansible.py

```diff
@@ -189,9 +234,15 @@ def start(self):
 
         try:
             while True:
+                nextCall = time.time()
                 event = sel.select()
                 self.udpReceiver()
                 self.unpackageData()
+                if (nextCall > time.time()):
+                    try:
+                        time.sleep(nextCall - time.time())
```

Comment:
Same comment as `ValueError` above. (No need for redundant `if` followed by try-catch. Single try-catch or what @yizow said would work)